app/presenters/profile.rb

```diff
@@ -77,6 +77,18 @@ def mentored_tracks
         user.track_mentor
       end
 
+      def can_contribute?
+        user_custom_contributions.any?
+      end
+
+      def contributions
```

Comment:
This name suggests that we are listing contributions that they have made, but it doesn't look like this is what it means.